mm2src/coins/nft/nft_errors.rs

```diff
@@ -98,7 +107,8 @@ impl HttpStatusCode for GetNftInfoError {
             | GetNftInfoError::Internal(_)
             | GetNftInfoError::GetEthAddressError(_)
             | GetNftInfoError::TokenNotFoundInWallet { .. }
-            | GetNftInfoError::DbError(_) => StatusCode::INTERNAL_SERVER_ERROR,
+            | GetNftInfoError::DbError(_)
+            | GetNftInfoError::ParseTimestampError(_) => StatusCode::INTERNAL_SERVER_ERROR,
```

Comment:
hmmm,  I put it as an internal problem, because if we get response from moralis successfully
```rust
let response = send_request_to_uri(uri.as_str()).await?;
```
other things if they fail will be our internal problems.

But may be you are right, if parsing from date str failed 
```rust
let date: DateTime<Utc> = date_str.parse()?;
```
the problem is in string and this string we get from moralis.

I will move `GetNftInfoError::ParseTimestampError(_)` to this part
`GetNftInfoError::InvalidResponse(_) => StatusCode::FAILED_DEPENDENCY,`

Reply:
Done